src/index.js

```diff
@@ -7,6 +7,7 @@ import Home from './Home';
 import Main from './examples/Main';
 import SearchBox from './examples/Searchbox';
 import Autocomplete from './examples/Autocomplete';
+import Heatmap from './examples/Heatmap';
```

Comment:
Can we align it like the rest? Sorry, OCD 😂

Reply:
not sure if I understood what you meant by align, so I sorted the import lines by length in https://github.com/google-map-react/google-map-react-examples/pull/19/commits/d9ff68f7d25dfd4e3e9bb3ce991aa7edfc39f357